src/main.c

```diff
@@ -640,9 +779,11 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
     //
     // NB: Starting with Windows 8.1, only single-event packets are traced.
     // This logic is here to support packet captures from older systems.
+    //
+    // This logic does not apply to events generated by Microsoft-Windows-Ras-NdisWanPacketCapture 
+    // which only used KW_SEND and KW_RECEIVE keywords
```

Comment:
"which only used"? Do you mean "which don't use"?

Also, add "NB:" to the beginning of your new note here.